app/build.gradle

```diff
@@ -64,14 +67,18 @@ dependencies {
     implementation "androidx.navigation:navigation-ui-ktx:$nav_version"
 
     // Firebase
-    implementation "com.google.firebase:firebase-messaging:20.0.1"
+    implementation "com.google.firebase:firebase-messaging:20.1.0"

```

Comment:
Suprised this compiled, previously we were getting an error from another library that was depended on the older version.  👌🏾